ebuild-writing/variables/text.xml

```diff
@@ -276,6 +276,15 @@ The following variables may or must be defined by every ebuild.
     <uri link="::ebuild-writing/eapi/#eapi=4">EAPI>=4</uri>.)
     </ti>
   </tr>
+  <tr>
+    <ti><c>HTML_DOCS</c></ti>
+    <ti>
+    An array or space-delimited list of documentation files for
+    the default src_install function to install using dodoc into
```

Comment:
einstalldocs is not mentioned (yet) in the devmanual, before I refer to that one, should I add that one to the install functions part? if yes, where? alphabetically after dosym?

dodoc -r, yes, I can add that.

And indeed, if developers are depending on the einstalldocs functionality, they don't need to care where the files are installed. I'll change that as well.

Depending on the answer for the einstalldocs, I will make a new commit.